init.lua

```diff
@@ -315,15 +326,19 @@ minetest.register_chatcommand("getrank", {
 	params = "<name> | name of player",
 	func = function(name, param)
 		if param and param ~= "" then
-			if minetest.get_player_by_name(param) then
-				local rank = ranks.get_rank(param) or "No rank"
+			if storage:get_string(param) then
+				local rank = storage:get_string(param):gsub("^%l", string.upper)
```

Comment:
`ranks.get_rank` should again be suitable. Also, what does the `:gsub` call do? Does it make the first character uppercase or something? Sorry, I'm not familiar with Lua's take on regex :sweat_smile: